<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStringBuilder</class>
 <widget class="QFrame" name="FormStringBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBoxBuilders">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>192</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Original name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Number</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text insertion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cryptographic Hash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Image Hash (Perceptual dHash)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="WidgetOriginalNameSetting" name="page"/>
       <widget class="WidgetNumberSetting" name="page_2"/>
       <widget class="WidgetReplaceSetting" name="page_3"/>
       <widget class="WidgetInsertTextSetting" name="page_4"/>
       <widget class="WidgetFileHashSetting" name="page_5"/>
       <widget class="WidgetImageHashSetting" name="page_6"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widgetButtons" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonRemove">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove this setting</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/res/images/x.svg</normaloff>:/res/images/x.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonUp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move up this setting</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/res/images/up.png</normaloff>:/res/images/up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDown">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Move down this setting</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/res/images/down.png</normaloff>:/res/images/down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>137</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetNumberSetting</class>
   <extends>QWidget</extends>
   <header>widgets/WidgetNumberSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetReplaceSetting</class>
   <extends>QWidget</extends>
   <header>widgets/WidgetReplaceSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetInsertTextSetting</class>
   <extends>QWidget</extends>
   <header>widgets/WidgetInsertTextSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetOriginalNameSetting</class>
   <extends>QWidget</extends>
   <header>widgets/WidgetOriginalNameSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetFileHashSetting</class>
   <extends>QWidget</extends>
   <header>widgets/WidgetFileHashSetting.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetImageHashSetting</class>
   <extends>QWidget</extends>
   <header>widgets/WidgetImageHashSetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBoxBuilders</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemove</sender>
   <signal>clicked()</signal>
   <receiver>FormStringBuilder</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
